api/hearings/models/hearings.enum.ts

```diff
@@ -6,7 +6,7 @@ export enum Actions {
 }
 
 export enum HMCStatus {
-  HEARING_REQUESTD = 'Hearing requested',
+  HEARING_REQUESTED = 'Hearing requested',
```

Comment:
thanks for spot this, could you also amend the one in src/../hearings.enum.ts

Reply:
😭 I was wondering what changed. So easy to miss.

Reply:
Thank you @rayliangatos I have now updated the code.